README.md

```diff
@@ -1,6 +1,27 @@
 # ansible-sumocollector
 ###### Ansible role to install SumoCollector. This role was inspired by modcloth's SumoCollector role, but it goes one step further by including the ability to include additional log paths.
 
+## Chromatic Specific Notes
```

Comment:
Can we write this in a way that is not specific to "Chromatic", but just details the functionality that is present/needed here?